src/uwtools/drivers/driver.py

```diff
@@ -307,6 +300,13 @@ def __init__(
 
     # Workflow tasks
 
+    @tasks
+    @abstractmethod
+    def provisioned_rundir(self):
+        """
+        Run directory provisioned with all required content.
+        """
+
```

Comment:
Relocated from the `Assets` base class to the `Driver` base class so that we could use the simplest driver in our tests.

Reply:
I'll add that it came up, in the context of the `CDEPS` driver, that some assets-only drivers may not have a single definition of "provisioned run directory", or even one that could be easily constructed with `if` statements. A user might just want to make multiple task calls on an assets-only driver to get what they need a la carte. But I think standalone drivers _can_ define a provisioned run directory: At the very least, it will contain a runscript. So this seems to me like a sound change.